### DISCUSSION TITLE
Login not possible after nightly update

### DISCUSSION THREAD

**Discussion by MartinM01** [nextcloud/all-in-one#1793] [Questions] [answered]
### Steps to reproduce
1. update aio
2.
3.

### Expected behavior 
Login possible

### Actual behavior 
Login not possible (PW wrong)
Background Picture of Loginscreen gone

### Host OS 
Ubuntu 20.04.5 LTS 

#### Nextcloud AIO version 
Nextcloud AIO v4.1.0

#### Current channel 
latest

**Comment by szaimen** [comment]
Hi, can you post a screenshot?

**Comment by MartinM01** [comment]
![image](https://user-images.githubusercontent.com/96419314/213120656-14bbb763-e3cf-4d2d-937b-16cf006cc658.png)

**Comment by MartinM01** [comment]
I tried starting from the backup, but it didn't change anything.

**Comment by szaimen** [comment]
Okay, that the background image might get lost during the upgrade to Nc25 is known (should be fixed with 25.0.3 in AIO 4.2.0), you need to reapply it in the theming settings. However login should still be possible. Can you try to delete all cookies for the website and checck if the login works afterwards?

**Comment by MartinM01** [comment]
Nextcloud Talk was possible with running sessions.

**Comment by MartinM01** [comment]
ha ok that works

**Comment by szaimen** [comment, marked as answer]
> ha ok that works

great! The fix for that is in https://github.com/nextcloud/server/pull/35419 but not merged yet, I hope it will be included in 25.0.4

**Comment by MartinM01** [comment]
Thanks for the quick solution and feedback
